src/main/kotlin/no/nav/syfo/sykmelding/service/EgenmeldtSykmeldingService.kt

```diff
@@ -97,9 +98,9 @@ class EgenmeldtSykmeldingService @KtorExperimentalAPI constructor(
             log.warn("Egenmeldt sykmelding kan ikke være mer enn {} dager", maxAntallDagerSykmeldt)
             throw ForLangPeriodeException("Egenmeldt sykmelding kan ikke være lenger enn $maxAntallDagerSykmeldt dager")
         }
```

Comment:
Vi er vel interessert i å sjekke om det finnes egenmeldinger de siste fire månedene..? Sånn sett må vi vel sjekke om det finnes egenmeldinger med periode mellom now().minusMonths(4) og now()?

Reply:
Det kan forresten være lurt å navngi argumentene til metoden når du har flere parametre med samme type :)

Reply:
Ja. I praksis er det allerede det vi gjør siden det ligger en regel inne som sørger for at fom alltid er dagens dato på linje 93, men jeg kan bytte ut fom med LocalDate.now() for å gjøre det tydligere :) 

Godt tips om å bruke navngitte argumenter i metodekall for å øke lesbarhet!